### ISSUE TITLE
Webtop Alpine in Home Assistant blank screen...

### ISSUE THREAD

**Issue by Andersonong-github** [alexbelgium/hassio-addons#939]
**Webtop Alpine KDE** in Home Assistant X86-6

name: Anderson
about: Webtop Alpine KDE is blackout screen after I installed in my HA

Hi,  Appology if I'm asking anything ain't right or stupid here.. just I'm facing problem on Webtop Alpine KDE in HA. I'm facing a problem that after done installed & launch the apps are just blackout only, anybody could help me on this cause I'm seek arround google whoever have similar problem about this but also cannot fix mine.. browser without plugin, clear cache & etc... 

I uninstall & reinstall 3 times, result still the same as picture attached.. just blackout with 1 tiny round circle with pop out to access virtual keyboard & etc.... thanks in advance

Home assistant - Webtop Alpine KDE

Thanks in advance & apprieciated🙏🙏🙏

**Comment by alexbelgium** [comment]
Thanks! The add-on is broken for the moment but I'm trying to fix it

**Comment by Andersonong-github** [comment]
Thanks and apprieciated. Excited the new batch launch😊

**Comment by alexbelgium** [comment]
:-) don't be too excited tough ; currently according to my test I managed to start the container manually (going in the ssh terminal addon and executing `docker run --rm -it --security-opt seccomp=unconfined --shm-size="1gb" -p 3001:3001 lscr.io/linuxserver/webtop:latest bash`) but many images just didn't start. I think there was 1 or 2 that worked (I think it was ubuntu-xfce) I will have to check again

**Comment by alexbelgium** [comment]
Mmh, there is an error `mkdir: cannot create directory '/sys/fs/cgroup/init': Read-only file system`

**Comment by alexbelgium** [comment]
To be updated : 
[] Black screen : alpine-xfce ; alpine-kde
[] Partially working : ubunbu-xfce
[] Ingress to solve

**Comment by alexbelgium** [comment]
Ubuntu works (without ingress) but graphics are messed up ; seems like an issue with drivers

**Comment by alexbelgium** [comment]
Ok it works. Ubuntu only it seems. So what do you want, KDE or XCFE ?
It seems XCFE is better with low memory systems

**Comment by Andersonong-github** [comment]
Thanks & apprieciated. As per your suggest XCFE.😊

**Comment by alexbelgium** [comment]
Next step is for me to test :
- [ ] TZ setting
- [ ] Keyboard setting
- [ ] Ingress

For me on rpi4 the default graphic device was not detected so I've implemented an addon option DRINODE where you need to specify you dri device (for me /dev/dri/card1)

**Comment by Andersonong-github** [comment]
I'm using X86/64 Intel NUC 7. Okay, I will try it out on dri device.

**Comment by Andersonong-github** [comment]
![Screenshot 2023-09-03 154219](https://github.com/alexbelgium/hassio-addons/assets/69752617/e1da9690-e813-4829-90ba-fae831c3d599)

**Comment by Andersonong-github** [comment]
virtual keyboard not able to use and just allow me to change position only..

**Comment by alexbelgium** [comment]
Hi, that's normal ingress does not work yet you need to enable the port in addon options and connect with ip:port

**Comment by Andersonong-github** [comment]
![Screenshot 2023-09-03 155818](https://github.com/alexbelgium/hassio-addons/assets/69752617/b0597162-1746-4b53-800d-b53ea2e56c26)

**Comment by Andersonong-github** [comment]
😅😅 Hi. IP:3000 TZ setting as UTC +8.

**Comment by alexbelgium** [comment]
Thanks, so kasmvnc connects but not webtop. I'll check thanks

**Comment by Andersonong-github** [comment]
![WhatsApp Image 2023-09-03 at 16 04 51](https://github.com/alexbelgium/hassio-addons/assets/69752617/2c42cc6f-8cb1-42a2-9282-68c950e871e4)

**Comment by alexbelgium** [comment]
Could you please try with puid guid 0 ?

**Comment by Andersonong-github** [comment]
![Screenshot 2023-09-03 170854](https://github.com/alexbelgium/hassio-addons/assets/69752617/2b6086ed-7a81-4344-9d29-2811bfd83ebe)

**Comment by Andersonong-github** [comment]
Hi Sir, finally it's works. Thanks and appreciated.
yes, PUID 0 & GUID 0 works now.

**Comment by Andersonong-github** [comment]
but virtual keyboard seem not working yet.

**Comment by alexbelgium** [comment]
Thanks for the feedback. I've added a chmod -R 755 for the home folder that seems now to allow to use PUID/GUID 1000
I've removed ingress to avoid confusion as it is not working yet
I'll check virtual keyboard but not sure how I can act on it from the addon

**Comment by alexbelgium** [comment]
Where do you find the virtual keyboard? I don't see it but can use my smartphone or device keyboard. Thanks

**Comment by Andersonong-github** [comment]
![Screenshot 2023-09-03 175743](https://github.com/alexbelgium/hassio-addons/assets/69752617/7ec231d4-f013-4a8c-beb2-4170ee4ef820)

**Comment by Andersonong-github** [comment]
I tried now can already. I found only touch screen is works well, if I hover my mouse over the icon & not able to toggle the virtual keyboard pop out. it has to be touch screen to toggle it pop out.

**Comment by alexbelgium** [comment]
Ingress works but when connecting on computer, phone or ipad my virtual keyboard works...

**Comment by Andersonong-github** [comment]
![Screenshot_20230905_142819_Home Assistant](https://github.com/alexbelgium/hassio-addons/assets/69752617/328eeed8-7c1a-4968-a1b6-fd9d1ceb9c1a)
![Screenshot_20230905_142807_Home Assistant](https://github.com/alexbelgium/hassio-addons/assets/69752617/e336cfba-cff9-40c4-9969-a8a68133db04)

**Comment by Andersonong-github** [comment]
Hi Sir, thanks for your works & apprieciated. After latest update push I'm not able to start the webtop.
Also last test with the previous version, it causing my HA down & I have to force hard reboot the whole machine. 
![Uploading Screenshot_20230903_182939_Home Assistant.jpg…]()

**Comment by Andersonong-github** [comment]
![Uploading Screenshot_20230903_182939_Home Assistant.jpg…]()

**Comment by Andersonong-github** [comment]
![Uploading Screenshot_20230903_182939_Home Assistant.jpg…]()

**Comment by alexbelgium** [comment]
Hi, TZ must be in the form Country/City (`Europe/Paris`), so I've added a text in the Readme to describe that. Here are valid values : https://manpages.ubuntu.com/manpages/focal/man3/DateTime::TimeZone::Catalog.3pm.html

I'll reinstall and test it but yesterday it worked fine on mine...

**Comment by Andersonong-github** [comment]
Alright, thanks sir.

Yes successful started TZ as Europe/Paris

**Comment by alexbelgium** [comment]
So everything is working ?

**Comment by Andersonong-github** [comment]
Yes sir, seem good now and thanks. I'm trying now & keep observe the HA supervisor😊

**Comment by alexbelgium** [comment]
Hi, this is not really an issue ; it's just a bug from HA. When there is devices, it allows reaching sda devices (hard drives). On virtual machines (which I guess you are on) or some systems where those devices don't exist, it just puts this error in the log. However, there is no issue linked to it, and nothing I can do as it is due to HA supervisor

**Comment by Andersonong-github** [comment]
I see. Okay², thank you very much sir😊😊

**Comment by Andersonong-github** [comment]
Hi Sir, may I know does this webtop able to install any application? I tried to installed few but kept failure, noted that it will causing the RAM consumption increase but just would like to know whether it is possible or not to install side loaded application.😊😅

**Comment by alexbelgium** [comment]
Hi, you can just add them in the addon options next to `engrampa`, but they must be part of the Ubuntu store. For example, `engrampa,libreoffice`
